$NetBSD: patch-ee,v 1.3 2010/10/27 13:24:09 tsutsui Exp $

Remove unnecessary includes. "qemu_socket.h" is enough and
including <netinet/tcp.h> without <netinet/in.h> causes errors.
(fixed in the upstream master but not in stable-0.13 branch)
http://git.qemu.org/qemu.git/commit/?id=aab2e8f79ad253c760787ff3ce4d64967fed0003

--- block/sheepdog.c.orig	2010-07-28 01:55:42.000000000 +0000
+++ block/sheepdog.c	2010-08-08 09:17:28.000000000 +0000
@@ -8,16 +8,6 @@
  * You should have received a copy of the GNU General Public License
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  */
-#ifdef _WIN32
-#include <windows.h>
-#include <winsock2.h>
-#include <ws2tcpip.h>
-#else
-#include <netdb.h>
-#include <netinet/tcp.h>
-
-#define closesocket(s) close(s)
-#endif
 
 #include "qemu-common.h"
 #include "qemu-error.h"
